luigi/rpc.py

```diff
@@ -76,6 +76,26 @@ def fetch(self, full_url, body, timeout):
         return urlopen(full_url, body, timeout).read().decode('utf-8')
 
 
+class BobaPKIHTTPAdapter(requests.adapters.HTTPAdapter):
+    """HTTP adapter which disables hostname validation on HTTPS connections.
+
+    Copied from $ATT/rpc2/affirm/rpc2/transport/http.py
+    """
+
+    def init_poolmanager(self, *args, **kwargs):
+        super(BobaPKIHTTPAdapter, self).init_poolmanager(assert_hostname=False, *args, **kwargs)
+
+
+def get_requests_session():
+    session = requests.Session()
+    cert_verify = os.environ.get('BOBAPKI_CACERT_VERIFY', None)
+    if cert_verify is not None:
+        session.mount('https://', BobaPKIHTTPAdapter())
+        session.verify = cert_verify
```

Comment:
Is this supposed to be a bool?

Reply:
it can be a string or boolean; but yeah we are passing a string here which is a path to the cert file